### PR TITLE
Author fields on annotaion; BICAN ext no additional props

### PR DESCRIPTION
**Pull request by dosumis**

### CODE REVIEW

File: general_schema.json [hunk 1]
```diff
@@ -128,6 +128,10 @@
           "items": {
             "type": "string",
             "description": "List of synonyms"
+          },
+          "author_annotation_fields": {
```
Comment (hkir-dev) [inline general_schema.json:132]: seems nesting is wrong, this emergent is inside `synonyms`.

Also, Should this be an array of objects? Such as:
`
"author_annotation_fields": {
            "type": "array",
            "description": "A set of author defined key value pairs annotating the cell set. The names and aims of these fields MUST not clash with annotationThis schema accepts author defined fields.",
            "items": {
                  "type": "object",
                  "description": "Author defined key value pairs annotating the cell set. The names and aims of these fields MUST not clash with annotationThis schema accepts author defined fields."
             }
          }
`

**Comment by hkir-dev** [comment]
Seems the example data `[Yao_ABC_labelset.json](https://github.com/cellannotation/cell-annotation-schema/pull/109#diff-1cad120ff7e6f221c534cb729e11703d35f451d5d916823a564314133a52331d)` wasn't in synch with the schema, updated it accordingly.

**Comment by hkir-dev** [comment]
#rollschema

**Comment by dosumis** [comment]
Have updated.

**Review by hkir-dev — APPROVED** [review]

**Comment by hkir-dev** [comment]
#rollschema

**Comment by hkir-dev** [comment]
Can we merge this PR? We are currently using a non-standard properties to support this and this is making linkml driven operations harder.

**Pull request merged**